scripts/deploy-helm.sh

```diff
@@ -263,7 +250,11 @@ else
   exit 1
 fi
 
-  check_proxy_settings
+if [ -n "$USE_PROXY" ]; then
+  if [ -z "$HTTP_PROXY" ] || [ -z "$HTTPS_PROXY" ]; then
+    echo -e "\e[91mERROR: \e[37mThe useProxy is set, please fill httpProxy or httpsProxy in your values file"
```

Comment:
should we continue deployment with this error? or should we exit in case of incorrect config?

Reply:
hmmmm, as you think. From my perspective if you not set url it will work without proxy